src/app/Http/Controllers/UploadController.php

```diff
@@ -25,40 +25,39 @@
 class UploadController extends Controller
 {
     /**
-    * Called when /add_user gets a post request from the forms.
-    */
-    public function upload() {
-
-      global $count_message;
-      global $add_user_message;
-      global $count_msg_with_duplicates;
-      global $count_msg_no_new;
-      global $add_user_err_message;
-
-      if (request()->has('mycsv')) {
-        $this->uploadFile();
-        return redirect()->route('add_user_route')
-          ->with('user_count_message', $count_message)
-          ->with('count_message_duplicates', $count_msg_with_duplicates)
-          ->with('count_message_no_new', $count_msg_no_new);
-      }
-      else if (request()->has('add_single_user')) {
-        $this->uploadUser();
-        return redirect()->route('add_user_route')
-        ->with('add_message', $add_user_message)
-        ->with('add_message_error', $add_user_err_message);
-      }
-      else {
-        return redirect()->route('add_user_route');
-      }
+     * Called when /add_user gets a post request from the forms.
+     */
+    public function upload()
+    {
+        global $count_message;
+        global $add_user_message;
+        global $count_msg_with_duplicates;
+        global $count_msg_no_new;
+        global $add_user_err_message;
+
+        if (request()->has('mycsv')) {
+            $this->uploadFile();
+            return redirect()->route('add_user_route')
+                ->with('user_count_message', $count_message)
+                ->with('count_message_duplicates', $count_msg_with_duplicates)
+                ->with('count_message_no_new', $count_msg_no_new);
+        } else if (request()->has('add_single_user')) {
+            $this->uploadUser();
+            return redirect()->route('add_user_route')
+                ->with('add_message', $add_user_message)
+                ->with('add_message_error', $add_user_err_message);
+        } else {
+            return redirect()->route('add_user_route');
+        }
```

Comment:
The majority of the changes in this file (UploadController.php) are caused by the formatter.